packages/keystone/README.md

```diff
@@ -6,14 +6,41 @@ order: 1
 
 # keystone
 
+## Constructor
+
+### Usage
+
+```javascript
+const keystone = new Keystone({
+  /*...config */
+});
+```
+
+### Config
+
+| Option                  | Type       | Default    | Description                                                                                                                                                 |
+| ----------------------- | ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `defaultAccess`         | `Object`   | `{}`       | FOO                                                                                                                                                         |
+| `adapters`              | `Array`    | `[]`       | FOO                                                                                                                                                         |
+| `adapter`               | `Object`   | `null`     | Required. The database storage adapter. See the [Adapter Framework](https://v5.keystonejs.com/keystone-alpha/keystone/lib/adapters/) page for more details. |
```

Comment:
`null` is not really the default so I suggest Moving  required into the default col similar to other docs:

```suggestion
| `adapter`               | `Object`   |  Required     | The database storage adapter. See the [Adapter Framework](https://v5.keystonejs.com/keystone-alpha/keystone/lib/adapters/) page for more details. |
```